gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinDriverIntegrateTest.java

```diff
@@ -102,6 +102,10 @@ public Settings overrideSettings(final Settings settings) {
                 deleteDirectory(new File("/tmp/neo4j"));
                 settings.graphs.put("graph", "conf/neo4j-empty.properties");
                 break;
+            case "shouldExecuteInSessionAndSessionlessWithoutOpeningTransaction":
```

Comment:
@PommeVerte note that you can fold this up into the `case` above as the neo4j initialization code is the same.  basically:

```java
case "shouldExecuteScriptInSessionOnTransactionalGraph":
case "shouldExecuteSessionlessScriptOnTransactionalGraph":
case "shouldExecuteScriptInSessionOnTransactionalWithManualTransactionsGraph":
case "shouldExecuteInSessionAndSessionlessWithoutOpeningTransaction":
    deleteDirectory(new File("/tmp/neo4j"));
    settings.graphs.put("graph", "conf/neo4j-empty.properties");
    break;
````